app/models/manageiq/providers/ansible_tower/provider.rb

```diff
@@ -60,6 +62,18 @@ def url=(new_url)
     default_endpoint.url = uri.to_s
   end
 
+  def self.seed
```

Comment:
how about seeding via the rest api? as a post install script. Or maybe something with the appliance console? This could be re-used to seed other providers too